test/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests/Visitors/DictionaryObjectTypeVisitorTests.cs

```diff
@@ -43,6 +44,7 @@ public void Given_Type_When_IsNavigatable_Invoked_Then_It_Should_Return_Result(T
         }
 
         [DataTestMethod]
+        [DataRow(typeof(Dictionary<object, string[]>), true)]
```

Comment:
In the OpenAPI spec, the Dictionary type is always Dictionary<string, ....>. It can't be Dictionary<object, ...>.